recipes/pymmcore/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pymmcore" %}
+{% set version = "10.1.1.70.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymmcore-{{ version }}.tar.gz
+  sha256: 7cb9ed0aac35ce4dbaaff8b967d41205bc98b15c621b931d4a2a75474c6d37b0
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - numpy >=1.12.0
+    - pip
+    - python
+  run:
+    - numpy >=1.12.0
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - pymmcore
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/micro-manager/pymmcore
+  summary: Python bindings for MMCore, Micro-Manager's device control layer
+  license: LGPL-2.1
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - tlambert03
```

```suggestion
    - tlambert03
    - marktsuchida
```

@marktsuchida, would you be willing to be listed as a maintainer on this recipe?

---

Yes. I don't know too much about Conda packaging, but should be able to help with pymmcore build issues.